fastlane/Fastfile

```diff
@@ -42,11 +51,102 @@ Dotenv.load(USER_ENV_FILE_PATH)
 ENV[GHHELPER_REPO = 'wordpress-mobile/WordPress-Android']
 ENV['PROJECT_ROOT_FOLDER'] = File.dirname(File.expand_path(__dir__)) + '/'
 ENV['FL_RELEASE_TOOLKIT_DEFAULT_BRANCH'] = 'trunk'
+DEFAULT_BRANCH = 'trunk'
 REPOSITORY_NAME = 'WordPress-Android'
 
 # Used by Fastlane to work around the Google API random failures
 ENV['SUPPLY_UPLOAD_MAX_RETRIES'] = '5'
 
+#####################################################################################
+# Version Methods
+#####################################################################################
+
+def commit_version_bump
+  Fastlane::Helper::GitHelper.commit(
+    message: 'Bump version number',
+    files: VERSION_PROPERTIES_PATH
+  )
+end
+
+# Returns the current version name from `version.properties` without needing formatting or calculations
+def current_version_name
+  VERSION_FILE.read_version_name
+end
+
+# Returns the release version of the app in the format `1.2` or `1.2.3` if it is a hotfix
+#
+def current_release_version
+  # Read the current release version from the .xcconfig file and parse it into an AppVersion object
+  current_version = VERSION_FORMATTER.parse(VERSION_FILE.read_version_name)
+  # Return the formatted release version
+  VERSION_FORMATTER.release_version(current_version)
+end
+
+#  Returns the next release version of the app in the format `1.2` or `1.2.3` if it is a hotfix
+#
+def next_release_version
+  # Read the current release version from the .xcconfig file and parse it into an AppVersion object
+  current_version = VERSION_FORMATTER.parse(VERSION_FILE.read_version_name)
+  # Calculate the next release version
+  next_release_version = VERSION_CALCULATOR.next_release_version(version: current_version)
+  # Return the formatted release version
+  VERSION_FORMATTER.release_version(next_release_version)
+end
+
+# Returns the beta version of the app in the format `1.2-rc-1`
+#
+def current_beta_version
+  # Read the current release version from the .xcconfig file and parse it into an AppVersion object
+  current_version = VERSION_FORMATTER.parse(VERSION_FILE.read_version_name)
+  # Return the formatted release version
+  VERSION_FORMATTER.beta_version(current_version)
+end
+
+# Returns the beta version that is used by the code freeze
+# It first increments the minor number, which also resets the build number to 0
+# It then bumps the build number so the -rc-1 can be appended to the code freeze version
+def first_beta_version
+  # Read the current release version from the .xcconfig file and parse it into an AppVersion object
+  current_version = VERSION_FORMATTER.parse(VERSION_FILE.read_version_name)
+  # Calculate the next major version number
+  next_minor_version = VERSION_CALCULATOR.next_minor_version(version: current_version)
+  # Calculate the next build number
+  first_beta_version = VERSION_CALCULATOR.next_build_number(version: first_beta_version)
```

Comment:
After the change in 08c57ebfd46cef2dc7a0f4c8943fc58abc5f0d86, I _think_ this needs to change like so:

```suggestion
  first_beta_version = VERSION_CALCULATOR.next_build_number(version: next_minor_version)
```

Reply:
Good catch, thank you!